samples/CallWithChat/CHANGELOG.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
We don't track changelog files for the samples.

How did these get created?

Reply:
I have been wondering that myself. @PorterNan  @JamesBurnside any idea?

Reply:
Beachball just creates them when other packages (that are downstream deps) are bumped, their only contents are about other packages being bumped